hibernate-core/src/main/java/org/hibernate/integrator/internal/IntegratorServiceImpl.java

```diff
@@ -40,7 +42,7 @@ public IntegratorServiceImpl(LinkedHashSet<Integrator> providedIntegrators, Clas
 		}
 	}
 
-	private void addIntegrator(Integrator integrator) {
+	private void addIntegrator(@UnknownInitialization IntegratorServiceImpl this, Integrator integrator) {
```

Comment:
Solve through static factory instead.

Reply:
Missed this one; one of the rare cases where that annotation didn't multiply ...